### ISSUE TITLE
Please update to work with the current version of PyCharm

### ISSUE THREAD

**Issue by JackAtOmenApps** [JetBrains/requirements#111]
**Describe the bug**
The plugin has no effect. Upon IDE startup, the out-of-date requirements are highlighted momentarily, and then all highlighting ceases as if the plugin were not installed.

**To Reproduce**
1. Have Requirements plugin installed
2. Look at requirements file

**Expected behavior**
It should highlight out-of-date requirements, as it always did before.

**IDE (please complete the following information. See Main Menu -> Help -> About):**

- PyCharm 2023.3.2 (Professional Edition)
- Build #PY-233.13135.95, built on December 19, 2023
- Runtime version: 17.0.9+7-b1087.9 amd64
- VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
- Linux 6.2.0-39-generic
- GC: G1 Young Generation, G1 Old Generation
- Memory: 4096M
- Cores: 12
- Registry:
  -   debugger.new.tool.window.layout=true
  -   ide.experimental.ui=true
- Non-Bundled Plugins:
  -   com.github.copilot (1.4.5.4049)
  -   XPathView (233.11799.188)
  -   com.intellij.aqua (233.13135.104)
  -   DBN (3.4.2455.0)
  -   com.intellij.ml.llm (233.13135.103)
  -   mobi.hsz.idea.gitignore (4.5.2)
  -   ru.adelf.idea.dotenv (2023.3)
  -   com.leinardi.pycharm.pylint (0.16.2)
  -   ru.meanmail.plugin.requirements (2022.4.1)
  -   net.ashald.envfile (3.4.2)
- Current Desktop: ubuntu:GNOME

**Plugin (please complete the following information. See Main Menu -> File -> Settings -> Plugins):**
 - Version 2022.4.1

**Comment by Enase** [comment]
+1

**Comment by abionics** [comment]
+1

**Comment by ghost** [comment]
Are you guys going to fix the plugin for latest and if so what's the timeframe? Having a timeframe would reduce the uncertainty.

**Comment by jacklinke** [comment]
@Enase @abionics @bretlowery @Coverttrickster @eyk107

The functionality in this plugin is *mostly* replaced by [PyCharm's bundled requirements.txt tool](https://www.jetbrains.com/help/pycharm/managing-dependencies.html#create-requirements).

I say 'mostly', because it only supports a single requirements file per project.

See: https://youtrack.jetbrains.com/issue/PY-71358/Requirements-Plugin-needs-update-not-working-in-many-months

**Comment by Enase** [comment]
@jacklinke That's true, I use it instead, but I really would love to have highlight and suggestions features in `requirements.txt`. it speedups the process significantly.  it's the main reason why I sill need this plugin. Thanks in advance.

**Comment by JackAtOmenApps** [comment]
@Enase I agree wholeheartedly, but JetBrains needs to hear it. I recommend following the YouTrack link in my previous comment, and share your input on the bug tracker there. I don't think anyone at JetBrains is watching the issues here.

(Oops, posted from other account. My personal GitHub account is @jacklinke)

**Comment by Enase** [comment]
@jacklinke  [done](https://youtrack.jetbrains.com/issue/PY-71358/Requirements-Plugin-needs-update-not-working-in-many-months#focus=Comments-27-9512542.0-0)
